content/en/docs/setup/install/multicluster/primary-remote/index.md

```diff
@@ -128,6 +128,10 @@ $ istioctl x create-remote-secret \
 
 Save the address of `cluster1`’s east-west gateway.
 
+{{< tip >}}
```

Comment:
Why would we have a tip for just one specific platform type, and only for one of the 4 setup variations? If we need such a tip, it should be generic, for any platform without an external LB, and it should be either moved up to the before you begin section, or made into a boilerplate and included everywhere it applies.

cc @nmittler

Reply:
I think this is needed for a single network. for multinetwork it works fine without an external LB.  Different cloud vendors already provide external LB and this is just for local testing. I found a ref for using it with `kind` but not sure about `minikube`.

I will move this instruction to the top.

Reply:
@shamsher31 In terms of using a common message, Frank had mentioned turning it into a boilerplate. See https://preliminary.istio.io/latest/about/contribute/shortcodes/#use-boilerplate-text for that.

Reply:
@ericvn Thanks. I have created a boilerplate for the same.